tests/e2e/cucumber/steps/ui/links.ts

```diff
@@ -8,6 +8,7 @@ When(
   async function (this: World, stepUser: string, stepTable: DataTable) {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const linkObject = new objects.applicationFiles.Link({ page })
+    await page.reload()
```

Comment:
please don't use reload here. After given steps like `Given "Alice" creates the following files into personal space using API`  you can put step `And "Alice" opens the "files" app` which makes element visible